client/src/components/FactList/FactList.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { List, ListItem, ListItemIcon, ListItemText } from '@material-ui/core';
+import { CheckCircleOutline, ErrorOutlineOutlined } from '@material-ui/icons';
+
+const FactList = () => {
+  return (
+    <div>
+      <h3>What is a SMART&reg; Health Card?</h3>
+      <List>
+        <ListItem>
+          <ListItemIcon>
+            <CheckCircleOutline color="secondary"/>
+          </ListItemIcon>
+          <ListItemText primary="Holds important vaccination or lab report data." />
+        </ListItem>
+        <ListItem>
+          <ListItemIcon>
+            <CheckCircleOutline color="secondary"/>
+          </ListItemIcon>
+          <ListItemText primary="Can be scanned to verify that the information has not been tampered with." />
+        </ListItem>
+        <ListItem>
+          <ListItemIcon>
+            <ErrorOutlineOutlined color="error"/>
+          </ListItemIcon>
+          <ListItemText primary="Don't forget - scanning this QR code will share all of the information on your Vaccine Card, so make sure to only share with those you trust." />
```

Comment:
```suggestion
          <ListItemText primary={<div><b>Don't forget</b> - scanning this QR code will share all of the information on your Vaccine Card, so make sure to only share with those you trust.</div>}/>
```
Something like this should allow you to capitalize the "Don't forget" text like it is in the wireframe mockup